github/pull_requests.md

```diff
@@ -0,0 +1,23 @@
+# Pull Requests
+
+This page provides a guidelines on how to craft pull requests. The goal is to have semi-structure and checklist which make the pull request easier to review.
+
+## General Guidelines
+
+- Think about the reader while crafting a PR. Take time to split the work after prototyping.
+- Make the scope of the PR small.
+- Try to change things in as few packages as possible. If changes are needed in too many packages at once it's an indication that the PR scope is too broad or package design is wrong.
```

Comment:
```suggestion
- Try to change things in as few packages as possible. If changes are needed in too many 
  packages at once it's an indication that the PR scope is too broad or package design is wrong.
```

Reply:
I decided to not break lines for practical reasons - it's easier to add suggestions w/o dealing with line length. I'd keep it how it is unless it's breaking your heart.

Reply:
Disagree and commit.